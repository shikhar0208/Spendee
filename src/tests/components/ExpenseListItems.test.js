import React from "react";
import { shallow } from "enzyme";
import ExpenseListItems from "../../components/ExpenseListItems";
import expenses from "../fixtures/expenses";

test("should render ExpenseListItem correctly", () => {
  const wrapper = shallow(<ExpenseListItems {...expenses[1]} />);
  expect(wrapper).toMatchSnapshot();
});
    