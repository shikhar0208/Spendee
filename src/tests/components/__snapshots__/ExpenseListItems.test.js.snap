// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    â‚¹10,950.00
    -
    December 28th, 1969
  </p>
</div>
`;
